package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "developer-portal",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "docs:dev": "vitepress dev docs",
+    "docs:build": "vitepress build docs",
+    "docs:preview": "vitepress preview docs"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "vitepress": "^1.0.0-alpha.49",
+    "vue": "^3.2.47"
+  }
+}
```

Comment:
I would've a lot of updates for this file, should I just push my version? I would also rebuild the package-lock.json with node v19 as it will use the lockfile version 3 for that.